aws-mfa

```diff
@@ -37,13 +35,10 @@ def main():
     parser.add_argument(
         '--duration',
         type=int,
-        help="""The duration, in seconds, that the credentials should remain
-        valid. Acceptable durations for IAM user sessions range from 900
-        seconds (15 minutes) to 129600 seconds (36 hours), with 43200 seconds
-        (12 hours) as the default. Sessions for AWS account owners are
-        restricted to a maximum of 3600 seconds (one hour). If the duration
-        is longer than one hour, the session for AWS account owners defaults
-        to one hour.""")
+        help="""The duration, in seconds, indicating how long the temporary
```

Comment:
What do you think the help could be here:
A mix of both AWS help:
The duration, in seconds, that the credentials should remain
valid. Acceptable durations for IAM user sessions range from 900
seconds (15 minutes) to 129600 seconds (36 hours), with 43200 seconds
(12 hours) as the default. Sessions for AWS account owners are
restricted to a maximum of 3600 seconds (one hour). If the duration
is longer than one hour, the session for AWS account owners defaults
to one hour.
The duration, in seconds, of the role session. The value can range 
from 900 seconds (15 minutes) to 3600 seconds (1 hour). By default, 
the value is set to 3600 seconds.


The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) to 3600 seconds (1 hour). By default, the value is set to 3600 seconds.